services/servicees.js

```diff
@@ -1,33 +1,64 @@
 const db = require("./../database_connection");
+const express = require("express");
+const app = express();
+app.use(express.json());
+const port = process.env.PORT || 3000;
 
+app.listen(port, () => {
+    console.log(`listening on port ${port}....`);
+});
 
 const  getall= ()=>{
-    db.query("SELECT * FROM tasks;");
+    try{
+        app.get('/',(req,res)=>{
+            const response = db.main("SELECT * FROM tasks;");
+            res.status(200).json(response)
+        })
+    }catch(err){
+        console.error(err);
+    };
 };
 
```

Comment:
l condition matemchich ki nda5el id mch mawjoud check it out

Reply:
Any logs ?

Reply:
chma3netha

Reply:
Ma3neha matal3lekch error fel terminal wela heja ?

Reply:
la la

Reply:
Mela kifeh te5demch ? Mafhemtech

Reply:
/tasks/id supposé tjiblik task by id but if il id mch fil db supposé ta3tina status 204 w errors msg '204 no content" ki njarab b postman yraja3li empty list w status 200

Reply:
Ayh bien sur lazmek ta3mel el error enti wa7dek 
If ( result fer8a ) return res.status...

Reply:
that is what line 30 is about illi 3morha la da5let lil condition heki ahat ki na3ti id mch mawjoud